examples/default.py

```diff
@@ -36,10 +36,11 @@ def make_variant(max_path_length):
             reparameterize=True, # should always be True
             kl_lambda=.1, # weight on KL divergence term in encoder loss
             use_information_bottleneck=True, # False makes latent context deterministic
+            # train embedding source should choose from {'initial_pool', 'online_exploration_trajectories', 'online_on_policy_trajectories'}
             train_embedding_source='online_exploration_trajectories',
-            # embedding_source should be chosen from
-            # {'initial_pool', 'online_exploration_trajectories', 'online_on_policy_trajectories'}
-            eval_embedding_source='online_exploration_trajectories',
+            resample_z='trajectory', # how often to resample z at test-time
+            resample_z_train=max_path_length, # how often to resample z at training time
```

Comment:
Seems weird to have resample_z take value 'trajectory', and resample_z_train=max path length.
I think we should maintain a consistent structure for controlling how embeddings are done at train and test.

Reply:
I agree...my original plan was to have all resample at any arbitrary rate. I'm not sure how to handle episodes that terminate before `max_path_length` though. Should we have a special argument value that means "trajectory" however long that happens to be?

Reply:
We could just have it consistently resample by trajectory instead of max_path_length?

Reply:
done in `de1b5c3`